umbraco/Umbraco-CMS
# Netcore: Migrate SignalR to .net core

Migrated SignalR to the new .Net Core version and moved over the SignalR files and adapted them.

Steps to test: 
1. Enter preview of a content node 
2. Open developer content
3. Make a change in content node
4. Make sure that the console logs "Reloading."
